### DISCUSSION TITLE
2.0.0-20079 Release

### DISCUSSION THREAD

**Discussion by IacobIonut01** [IacobIonut01/Gallery#226] [Announcements]
## What's Changed
* Update agp to v8.2.0-rc02 by @renovate in https://github.com/IacobIonut01/Gallery/pull/216
* Update coilVersion to v2.5.0 by @renovate in https://github.com/IacobIonut01/Gallery/pull/217
* Added translations from Crowdin
* Monthly timeline should be fixed now
* Added fast-scroll on timeline

### Update is live on Google Play

**Full Changelog**: https://github.com/IacobIonut01/Gallery/compare/2.0.0-20070...2.0.0-20079

<hr /><em>This discussion was created from the release <a href='https://github.com/IacobIonut01/Gallery/releases/tag/2.0.0-20079'>2.0.0-20079 Release</a>.</em>

**Comment by Bardock88** [comment]
Sorry for the question but why is Android 11 the minimum version for installation and not previous versions? How Android 5.x or 8.x?

**Reply by IacobIonut01** [reply]
Because of when certain APIs methods were introduced in MediaStore (some essential features like Trash are added with android 11)

**Reply by IacobIonut01** [reply]
This is the minimum android version compatible without removing some of the features of the app

**Comment by pablo03v** [comment]
I just wanna say thank you for this app. It works so freaking well and since the last time I used it, its design got so much better again!

**Reply by IacobIonut01** [reply]
Thanks for the kind words!

New release coming later today!

**Comment by WSTxda** [comment]
This is insane and amazing. What a cool app!
